### PR TITLE
[exampleApp] remove RichContent legacy, use only Ricos components

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/ricos-common/web/src/RicosTypes.ts [hunk 1]
```diff
@@ -64,7 +64,9 @@ export interface RicosEditorProps extends RicosProps {
   /* Changes to this interface should also be reflected in the API docs */
 }
 
-export interface RicosViewerProps extends RicosProps, Pick<RichContentViewerProps, 'addAnchors'> {
+export interface RicosViewerProps
+  extends RicosProps,
+    Pick<RichContentViewerProps, 'addAnchors' | 'experiments'> {
```
[the remark this answers is not in the document]
Comment (mharsat) [reply packages/ricos-common/web/src/RicosTypes.ts:69]: Wait is this actually supported directly on `RicosViewer`?

**Review by mharsat — CHANGES_REQUESTED** [review]
Amazing, we lost some extra pounds there!

File: packages/ricos-common/web/src/RicosTypes.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/ricos-common/web/src/RicosTypes.ts:69]: No, experiments should passed trough _rcProps, fixed it.

File: examples/main/shared/editor/Editor.tsx [hunk 2]
```diff
@@ -232,113 +141,43 @@ export default class Editor extends PureComponent<ExampleEditorProps, ExampleEdi
...
+            ref={ref => (this.editor = ref)}
+            onChange={onRicosEditorChange}
+            content={contentState}
+            injectedContent={injectedContent}
+            linkSettings={{ anchorTarget, relValue }}
+            locale={locale}
+            cssOverride={theme}
+            toolbarSettings={{
+              useStaticTextToolbar: textToolbarType === 'static',
+              getToolbarSettings: this.config.getToolbarSettings,
```
[the remark this answers is not in the document]
Comment (maimonav) [reply examples/main/shared/editor/Editor.tsx:170]: `getToolbarSettings` is being used to in rendering the toolbars.
`uiSettings` is being used also, but we can remove it's so important.

[the remark this answers is not in the document]
Comment (mharsat) [reply examples/main/shared/editor/Editor.tsx]: I didn't explain myself well, I meant we can probably remove a lot of code from `EditorPlugins` & `ViewerPlugins` and simplify things

[the remark this answers is not in the document]
Comment (maimonav) [reply examples/main/shared/editor/Editor.tsx]: Yeah probably, but due to more urgent tasks, I will merge this for now, we'll get to it, someday :)

**Comment by maimonav** [comment]
@mharsat Anything else?

**Review by mharsat — APPROVED** [review]

**Pull request merged**